### DISCUSSION TITLE
How push a github code space in a repositories

### DISCUSSION THREAD

**Discussion by wilonweb** [community/community#40845] [Codespaces] [answered]
### Select Topic Area

How push a github code space project. 

### Body

Hello 
I make a github codespace with Create-react-app i config some components, the app works on but i push the code, and i make branch and my code does not in my repo. 
Do you know what i can do ?
This is my project
[PianoTheory](https://github.com/NinjaTurtles-cloud/PianoTheory )

**Comment by wilonweb** [comment, marked as answer]
Is Okay !!! 
I had to use git bash ^^

**Comment by otter-computer** [comment]
Hi @NinjaTurtles-cloud, welcome to the GitHub Community!

It sounds like you still need to commit and push your changes to your repository after you've made changes in your codespace.

First you'll need to [create a commit](https://docs.github.com/en/codespaces/developing-in-codespaces/using-source-control-in-your-codespace#committing-your-changes) of all the changes you've made, then you'll need to [push](https://docs.github.com/en/codespaces/developing-in-codespaces/using-source-control-in-your-codespace#pushing-changes-to-your-remote-repository) them to your repository before you'll see them on github.com.

**Reply by wilonweb** [reply]
Thansk at the end of the afternoon i understood how its works ^^
